libraries/lib-time-and-pitch/StaffPadTimeAndPitch.cpp

```diff
@@ -127,6 +138,10 @@ bool StaffPadTimeAndPitch::SetRatio(
    {
       return false;
    }
+   if (r == member)
+   {
+      return true;
```

Comment:
Should we return `true` here? If I remember correctly - this will force `SetTimeAndPitch` to reboot anyway. Chances are I am wrong :-)

Reply:
I apologize for having wasted your time, but I just pushed [f734716e7](https://github.com/audacity/audacity/pull/4701/commits/f734716e7e9d4a8f6eec842f0cb7b2ea9bf4343f), which removes the possibility of changing time or pitch ratios on the fly. The rationale behind is in the commit message.